src/components/deals/details/DealDetails.tsx

```diff
@@ -5,79 +5,67 @@ import TagsList from '../../forms/add-a-deal/TagsList'
 import ClickableCoupon from '../../ClickableCouponCode'
 import { format } from 'date-fns'
 import Image from 'next/image'
-import Section, { SECTION_STYLE, SECTION_WIDTH } from '../../Section'
-import ShareDealButton from './ShareDealButton'
-import ReportDealButton from './ReportDealButton'
+import Section, { SECTION_WIDTH } from '../../Section'
 
 interface DealDetailsProps {
   deal: DealWithTags
 }
 export default function DealDetails({ deal }: DealDetailsProps) {
   return (
     <div className="text-white">
-      <Section width={SECTION_WIDTH.SM} className="pb-32 sm:pb-40 md:pb-80">
-        <div className="mb-10">
-          <p className="mb-2 font-bold  text-teal-500">{deal.category || ''}</p>
-          <div className="mb-10 flex flex-row items-end justify-between gap-x-4">
-            <h1 className="inline text-2xl transition-colors hover:text-teal-500 md:text-3xl">
-              {deal.name}
-            </h1>
-            <a
-              href={`/api/tracking?id=${deal.xata_id}&url=${deal.link}`}
-              className="flex cursor-pointer items-center gap-x-2 rounded-md bg-teal-500 px-2 py-1 text-black transition-all hover:scale-105 hover:bg-teal-500/90"
-            >
-              <span>Visit</span>
-              <Image
-                src="/icons/upright-arrow.svg"
-                alt=""
-                width={16}
-                height={16}
-              />
-            </a>
-          </div>
-
-          <div className="text-md mb-4 font-light md:text-lg">
-            <div className="mb-1 flex flex-wrap gap-2">
-              <span className="w-40 text-white/70 ">Valid from:</span>
-              <span className="font-bold">
-                {`${format(new Date(deal.startDate), 'MMM d, yyyy')} - ${deal.endDate ? format(new Date(deal.endDate), 'MMM d, yyyy') : '(no end date)'}` ||
-                  'No coupon code required'}
-              </span>
-            </div>
-            <div className="mb-4 flex flex-wrap gap-2 text-lg font-light md:mt-1.5">
-              <span className="w-40 text-white/70">Coupon Code:</span>
-              <span className="font-bold">
-                {deal.coupon ?
-                  <ClickableCoupon coupon={deal.coupon} />
-                : 'No coupon code required'}
-              </span>
-            </div>
-            <TagsList tags={deal.tags} />
-          </div>
+      <div className="mb-10">
+        <p className="mb-2 font-bold  text-teal-500">{deal.category || ''}</p>
+        <div className="mb-10 flex flex-row items-end justify-between gap-x-4">
+          <h1 className="inline text-2xl transition-colors md:text-3xl">
+            {deal.name}
+          </h1>
+          <a
+            href={`/api/tracking?id=${deal.xata_id}&url=${deal.link}`}
+            className="flex cursor-pointer items-center gap-x-2 rounded-md bg-teal-500 px-2 py-1 text-black transition-all hover:scale-105 hover:bg-teal-500/90"
+          >
+            <span>Visit</span>
```

Comment:
I might make this more accessible by adding an `sr-only` `span`:

```tsx
<span>Visit</span>
<span className="sr-only">this deal</span>
```

For screen readers, I don't think "Visit" is clear enough. I'd love to know @EmmaDawsonDev's thoughts.

Reply:
Depends how many deals are on the page. If there are several and all have a visit button then they definitely need making more unique so you can tell the difference when you pull up a list of links

Reply:
This would be for an individual deal. Only one of these components would be on any given page so seems like visit will work for now. Thanks @ryandotfurrer and @EmmaDawsonDev !!

Reply:
@EmmaDawsonDev it doesn't hurt anything to add the additional bit of info though right?

Reply:
In this way, it's perfectly fine. It's always best to make sure the visible text is first so that if a voice control user says "click visit" it will still activate the button and that's what happens here.